tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/configuration/OpenPaasConfiguration.java

```diff
@@ -57,14 +60,17 @@ public static boolean isConfiguredContactConsumer(Configuration configuration) {
         return readRabbitMqUri(configuration).isPresent();
     }
 
-    public static Optional<AmqpUri> readRabbitMqUri(Configuration configuration) {
+    public static Optional<List<AmqpUri>> readRabbitMqUri(Configuration configuration) {
```

Comment:
We don't need the Optional wrapper here. An empty list should do.

Reply:
Feel free to open a follow up PR to clean this up.